packages/components/badge/src/BadgeItem.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import { makeVariants } from '@spark-ui/internal-utils'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { DEFAULT_INTENT, DEFAULT_SIZE, DEFAULT_TYPE } from './config'
+
+export const styles = cva(
+  ['inline-flex h-fit', 'empty:p-none', 'text-center font-bold', 'border-md rounded-full'],
+  {
+    variants: {
+      intent: makeVariants<'intent', ['error', 'info']>({
+        error: ['bg-error', 'text-on-error'],
+        info: ['bg-info', 'text-on-info'],
+      }),
+      size: makeVariants<'size', ['sm', 'md']>({
+        sm: ['text-small', 'px-[var(--sz-6)] py-[var(--sz-2)]', 'empty:h-sz-12 empty:w-sz-12'],
+        md: ['text-caption', 'px-md py-sm', 'empty:h-sz-16 empty:w-sz-16'],
+      }),
+      type: {
+        relative: ['-left-lg empty:-left-md relative'],
```

Comment:
I'm just wondering if the relative positionning shouldn't use 50% translate insted of arbitrary values (lg, md, etc). Because like you said, the width of the component can change depending on its contents (1, 99, 99+, etc)

WDYT ?